.github/workflows/kind.yml

```diff
@@ -127,12 +161,21 @@ jobs:
   # yet.
   artifact-cleanup:
     name: Delete uploaded images
-    needs: [build-policy-recommendation-image, test-e2e-encap]
-    if: ${{ always() && needs.build-policy-recommendation-image.result == 'success' }}
+    needs:
+    - build-policy-recommendation-image
+    - build-clickhouse-monitor-image
+    - test-e2e-encap
+    - test-upgrade-from-N-1
+    if: ${{ always() && (needs.build-policy-recommendation-image.result == 'success' || needs.build-clickhouse-monitor-image.result == 'success') }}
```

Comment:
Looks like Quan has made an update on the Antrea side that:
`artifact-cleanup job now runs only when all test jobs succeed, to be
able to re-run failed jobs without re-running the image building job
and the succeeded jobs that depend on it. After all jobs succeed, the
cleanup job will still run, or the artifact will be deleted after
retention days (currently set to 1 day).`
I think we could follow this change in Theia.
Ref: https://github.com/antrea-io/antrea/pull/3985

Reply:
Make sense to me. Updated, thanks!